Feature/safety exit

sookim914 opened the pull request:
## Description

<!-- Describe your changes here -->

- Asana ticket:

## PR Checklist

<!-- Please validate your changes with the checklist below before marking for code review. -->

- [ ] Assign @romellogoodman as a reviewer.
- [ ] If your PR is not a hotfix, is it targeted for `dev`?
- [ ] Unit and functional test coverage was added where applicable.
- [ ] CI/CD passes for your PR.
- [ ] Complex code is well documented with comments.
- [ ] Does the original ticket have test instructions? If not add them below

## How to Test

<!-- Provide instructions for how to test/validate the changes. -->

phpsmith commented:
Added the logo to header - wasn't sure where classes will end up, so put an inline component style to limit size on logo for now.

sookim914 committed:
Add safety exit button

- Added safety exit button
- Removed `find resources` button
- replaced `find resources` button with safety exit button
in dropdown (MOBILE)

romellogoodman commented on package.json:
```diff
@@ -15,6 +15,7 @@
     "@material-ui/icons": "^4.9.1",
     "autosuggest-highlight": "^3.1.1",
     "axios": "^0.19.2",
+    "bootstrap": "^4.4.1",
```
Can you remove bootstrap and react-bootstrap. These dependencies aren't being used anywhere in the code

romellogoodman commented on src/App.scss:
```diff
@@ -72,6 +72,28 @@ input[class*='inputSingleline'] {
   }
 }
 
+.safetyExit {
```
This file seems to be generally for app wide styles. Can you add these styles to `src/components/NavDesktop.js` with the rest of the button's styles?

romellogoodman commented on src/components/NavDesktop.js:
```diff
@@ -8,6 +8,22 @@ import AccountNav from './AccountNav';
 import AsylumConnectButton from './AsylumConnectButton';
 import FavoritesLink from './FavoritesLink';
 import Language from './Language';
+import '../App.scss'
```
You can remove this import once the styles from that file are moved here. Also, I dont believe you have to keep importing the `App.scss` fine since its already imported in `App.js` and thus included throughout the app

romellogoodman commented on src/components/NavDesktop.js:
```diff
@@ -8,6 +8,22 @@ import AccountNav from './AccountNav';
 import Language from './Language';
+import '../App.scss'
+
+const transparentSpace = {
```
Can you add these styles and the ones from `App.scss` into the styles object below? This keeps styling consistent in the entire app. Here is the documentation for how to use that library https://material-ui.com/styles/basics/

sookim914 committed:
Removed unused dependencies

sookim914 committed:
removed unused dependencies

sookim914 committed:
moved media queries to navdesktop.js

sookim914 committed:
Edit styling to keep styling consistent

sookim914 committed:
Add styling to keep styling consistent
